Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ruby:2.4.1-alpine
+
+# Install system dependencies
+RUN apk --update add \
+      build-base \
+      mariadb-dev \
+      postgresql-dev \
+      sqlite-dev \
+      su-exec \
+      tini && \
+    gem update --system && \
+    gem update bundler && \
+    rm -rf /var/cache/apk/*
+
+# Create gemstash user
+ENV GEMSTASH_USER="gemstash"
+ENV GEMSTASH_HOME="/home/${GEMSTASH_USER}"
+RUN addgroup -g "9999" "${GEMSTASH_USER}"
+RUN adduser -S -D -u "9999" -G "${GEMSTASH_USER}" "${GEMSTASH_USER}"
```

Comment:
I think we should combine the `addgroup` and `adduser` into a single `RUN` statement.